tasks/jest/setup.js

```diff
@@ -9,6 +9,8 @@
 
 global.__DEV__ = true;
 
+jest.setTimeout(100000);
```

Comment:
```suggestion
```

Is there an alternative here that we could substitute? Is it just that DAP takes a while?

Reply:
It doesn't take _that long_ -- I was just troubleshooting and forgot to set it back. I'll dial it down. 👍

Reply:
It looks like with DAP testing loading-test.js it takes between 18 and 22 seconds. This could partly be because I'm on a Windows machine running WSL. 25000 should be good.

Reply:
Is there any chance we could colocate this with where it's called, or potentially call it from the matcher? Just curious